storage/storage.go

```diff
@@ -349,7 +349,7 @@ var (
 )
 
 // v2SanitizeHeaders applies the specifications for canonical extension headers at
-// https://cloud.google.com/storage/docs/access-control/signed-urls#about-canonical-extension-headers.
+// https://cloud.google.com/storage/docs/authentication/canonical-requests#about-headers.
```

Comment:
I don't think this is correct for `v2SantizeHeaders` because the link you add is for V4 signing only.

@frankyn any thoughts on what is the appropriate link here?

Reply:
Nice catch. I've updated it with this link, which looks like's it's for v2 urls: https://cloud.google.com/storage/docs/access-control/signed-urls-v2#about-canonical-extension-headers